### ISSUE TITLE
__init__() got multiple values for argument 'strides'

### ISSUE THREAD

**Issue by noiplcx** [tensorflow/tensorflow#16278]
model = Sequential()
    model.add(ZeroPadding2D((1, 1), input_shape=(img_width, img_height, 3)))
    print(model.output_shape)
    model.add(Convolution2D(64, 3, 3, strides=(
        1, 1), activation='relu', name='conv1_1'))
above is my code, I got error:
__init__() got multiple values for argument 'strides'
If i don't use 'strides', it's fine. but the stride is 3. How should I set strides?

**Comment by noiplcx** [comment]
from tensorflow.contrib.keras.python.keras.preprocessing.image import ImageDataGenerator, load_img, img_to_array
from tensorflow.contrib.keras.python.keras import optimizers
from tensorflow.contrib.keras.python.keras.models import Sequential
from tensorflow.contrib.keras.python.keras.layers.convolutional import Convolution2D, ZeroPadding2D
from tensorflow.contrib.keras.python.keras.layers.pooling import MaxPooling2D
from tensorflow.contrib.keras.python.keras.layers.core import Activation, Dropout, Flatten, Dense
from tensorflow.contrib.keras.python.keras import callbacks
from tensorflow.contrib.keras.python.keras import backend as K

The version of tensorflow is 1.1.0

**Comment by tatianashp** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
